modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -298,6 +297,12 @@ export class MessengerClient {
   }
 
   async sendAction(senderId: string, action: MessengerAction) {
+    const config = await this.getConfig()
+    if (config.disabledActions?.length && config.disabledActions.includes(action)) {
```

Comment:
Checking the length is not necessary, this is simpler: `config.disabledActions?.includes(action)`